src/components/Pivot/Pivot.md

```diff
@@ -0,0 +1,43 @@
+# Pivot
+Includes a Persona component with contact options.
+
+## Variants
+
+### Default
+@@include('Pivot.html')
+
+### Large
+@@include('Pivot.Large.html')
+
+### Tabs
```

Comment:
That's a good question for the designers. I'm not clear on when one is preferred over the other.